From 88d9ff76d7715193eafb881a0d572b16cdbdd824 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 28 Jun 2022 20:24:21 +0200
Subject: [PATCH 196/841] refactor(st-bsec): remove unused defines BSEC_MODE_

Remove the unused defines BSEC_MODE_OPEN1, OPEN2, SECURED, INVALID
in bsec API: bsec.h.

Use the existing BSEC_OTP_STATUS_INVALID to select the VALID bit.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git f54786457980c650de5c24f8eb689088fa3b76b3 refactor(st-bsec): remove unused defines BSEC_MODE_]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I8073e1bb4d1ba1a1db8e8e35a06388925c633bcf
---
 include/drivers/st/bsec.h | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/include/drivers/st/bsec.h b/include/drivers/st/bsec.h
index aae36c093..984825d3d 100644
--- a/include/drivers/st/bsec.h
+++ b/include/drivers/st/bsec.h
@@ -49,14 +49,6 @@
 
 #define BSEC_HARDWARE_KEY		BIT(8)
 
-/*
- * OTP MODE
- */
-#define BSEC_MODE_OPEN1			0x00U
-#define BSEC_MODE_SECURED		0x01U
-#define BSEC_MODE_OPEN2			0x02U
-#define BSEC_MODE_INVALID		0x04U
-
 /*
  * OTP Lock services definition.
  * Value must corresponding to the bit number in the register.
-- 
2.47.2

